cypress-tests/cypress/support/commands.js

```diff
@@ -961,23 +1021,25 @@ Cypress.Commands.add(
     }).then((response) => {
       logRequestId(response.headers["x-request-id"]);
       expect(response.headers["content-type"]).to.include("application/json");
-      if (response.status === 200) {
-        expect(response.body).to.have.property("currency");
-        if (resData["payment_methods"].length == 1) {
-          function getPaymentMethodType(obj) {
-            return obj["payment_methods"][0]["payment_method_types"][0][
-              "payment_method_type"
-            ];
+      cy.wrap(response).then(() => {
```

Comment:
move to `L1023` and:
```suggestion

      cy.wrap(response).then(() => {
```